Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.9
+RUN go get -u github.com/golang/dep/cmd/dep
+WORKDIR $GOPATH/src/github.com/grafana/grafana
+COPY Gopkg.toml Gopkg.lock ./
+RUN dep ensure --vendor-only
+COPY pkg pkg
+RUN go install ./pkg/cmd/grafana-server
+RUN go install ./pkg/cmd/grafana-cli
+RUN strip $GOPATH/bin/grafana-server
+RUN strip $GOPATH/bin/grafana-cli
+
+FROM node:8
+WORKDIR /usr/src/app/
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+ENV NODE_ENV production
+COPY . ./
```

Comment:
This should be able to copy just what's needed for the web build, avoiding vendor, pkg, etc.

Reply:
Would definitely be better. Do you have a list of things the Grunt process requires?

Reply:
I'm not sure off hand but maybe start with `public/, scripts/, Gruntfile.js, tsconfig.json, tslint.json` and add from there until the build succeeds.